docs/conditional.rst

```diff
@@ -26,113 +24,95 @@ Suppose that you have defined a `Thread` Model for the examples below.
         views = NumberAttribute(default=0)
 
 
-AND vs. OR
-^^^^^^^^^^
+Condition Expressions
+^^^^^^^^^^^^^^^^^^^^^
 
-Specifying that the conditions should be applied with AND or OR is achieved through the use of the `conditional_operator` keyword,
-which can be `and` or `or`.
+PynamoDB supports creating condition expressions from attributes using a simple syntax. Attributes can be compared against values.
```

Comment:
We should state that `we support conditional expressions by from attributes using Python's built-in comparison operators. For DynamoDB expressions that don't have a built-in operator, they are a function call.`

I like the code examples and we should keep them. But we should have a complete list where we describe the arguments it takes, as it is not quite as clear. If we created a table with the following columns, I think it would clear up a lot of things:
* the operator 
* a short description (optional as it might be self-explanatory).
* an example of use (how many arguments it takes, etc)
* the result

Here are some examples:
https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not
https://docs.python.org/2/library/stdtypes.html#comparisons
https://pandas.pydata.org/pandas-docs/stable/api.html#binary-operator-functions